package.json

```diff
@@ -5,32 +5,36 @@
   "scripts": {
     "start": "node node_modules/react-native/local-cli/cli.js start",
     "postinstall": "chmod +x ./bin/add-ios-source-maps.sh && ./bin/add-ios-source-maps.sh && chmod +x ./bin/add-ios-env-config.sh && ./bin/add-ios-env-config.sh",
-    "test": "jest",
+    "test": "jest --verbose",
     "prettify": "prettier --write 'ts/**/*.ts'",
     "packager:clear": "rm -rf $TMPDIR/react-native-packager-cache-* && rm -rf $TMPDIR/metro-bundler-cache-*",
     "bundle:android": "node node_modules/react-native/local-cli/cli.js bundle --platform android --dev true --entry-file index.js --bundle-output ./bundles/index.android.bundle",
     "tsc:noemit": "tsc --noemit",
     "lint": "tslint -p . -c tslint.json -t verbose",
-    "lint-autofix": "tslint -p . -c tslint.json -t verbose --fix"
+    "lint-autofix": "tslint -p . -c tslint.json -t verbose --fix",
+    "runandroid": "react-native run-android",
```

Comment:
we use `:` for subcommands (like `bundle:android` above) so there should be `run:android` and `run:ios`